docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -132,17 +132,19 @@ Type mapping
 ------------
 
 Both Oracle and Trino have types that are not supported by the Oracle
-connector. The following sections explain their type mapping.
+connector. Because Trino and Oracle each support types that the other does not, this
```

Comment:
Other connectors don't have the first sentence, "Both xxx and Trino ...". Do we want to remove it for consistency?

Reply:
We're using https://github.com/trinodb/trino/pull/13118 as a template for our type mapping fixups, which introduces that sentence. So the goal is instead to add it for consistency.

Reply:
#13118 doesn't contain the first sentence in this PR.

> Both Oracle and Trino have types that are not supported by the Oracle
connector.

Reply:
Ah, you're right. @tlblessing Can you please remove that first sentence?

Reply:
@ebyhr I agree that the sentence should be removed for consistency. At first, I thought it should remain but its nuance actually sounds confusing to me now. cc: @jhlodin